lib/css/components/_stacks-code-blocks.less

```
@@ -0,0 +1,102 @@
+pre[class*="language-"],
+pre.s-code-block {
+    font-size: @fs-body1;
+    font-family: @ff-mono;
+    line-height: @lh-lg;
+    color: var(--highlight-color);
+    background-color: var(--highlight-bg);
+    border-radius: @br-md;
+    max-height: 24rem;
+    margin: 0;
+    padding: @su8 @su12;
+    overflow: auto;
+
+    @scrollbar-styles();
+}
+
+code[class*="language-"] {
```

I suggest also supporting the case where `code` is bare, but the enclosing `pre` has the `s-code-block` class attached. This is going to be required for prod since we attach the language class to the `pre` instead and switching that to `code` is a major pain (I know, I tried...)

```suggestion
pre.s-code-block > code,
code[class*="language-"] {
```